set.mm

```diff
@@ -130235,8 +130235,9 @@ nonnegative integers (cont.)". $)
     $( Induction step. $)
     uzind.6 $e |- ( ( M e. ZZ /\ k e. ZZ /\ M <_ k ) -> ( ch -> th ) ) $.
     $( Induction on the upper integers that start at ` M ` .  The first four
-       hypotheses give us the substitution instances we need; the last two are
-       the basis and the induction step.  (Contributed by NM, 5-Jul-2005.) $)
+       hypotheses give us the substitution instances we need; the following two
```

Comment:
Here, we only have 6 hypotheses, so "the first four" and "the last two" was correct.